objectherkenning_openbare_ruimte/inference_pipeline/source/OORInference.py

```diff
@@ -0,0 +1,101 @@
+                target_classes: List
+                    List of target classes for which bounding boxes will be predicted.
+                sensitive_classes: List
+                    List of sensitive classes which will be blurred in output images.
+                target_classes_conf: Optional[float] = None
+                    Optional: confidence threshold for target classes. Only detections
+                    above this threshold will be considered. If omitted,
+                    inference_param["conf"] will be used.
+                sensitive_classes_conf: Optional[float] = None
+                    Optional: confidence threshold for sensitive classes. Only
+                    detections above this threshold will be considered. If omitted,
+                    inference_param["conf"] will be used.
+                output_image_size: Optional[Tuple[int, int]] = None
+                    Optional: output images will be resized to these (width, height)
+                    dimensions if set.
+                defisheye_flag: bool = False
+                    Whether or not to apply distortion correction to the input images.
+                defisheye_params: Dict = {}
+                    If defisheye_flag is True, these distortion correction parameters
+                    will be used. Contains "camera_matrix", "distortion_params", and
+                    "input_image_size" (size of images used to compute these
+                    parameters).
+                save_images: bool = False
+                    Whether or not to save the output images.
+                save_labels: bool = True
+                    Whether or not to save the annotation labels.
+                save_all_images: bool = False
+                    Whether to save all processed images (TRue) or only those containing
+                    objects belonging to one of the target classes (False).
+                save_images_subfolder: Optional[str] = None
+                    Optional: sub-folder in which to store output images.
+                save_labels_subfolder: Optional[str] = None
+                    Optional: sub-folder in which to store annotation labels.
+                batch_size: int = 1
+                    Batch size for inference.
+        defisheye_flag: bool = False
+            Whether or not to apply distortion correction to the input images.
+        defisheye_params: Dict = {}
+            If defisheye_flag is True, these distortion correction parameters
+            will be used. Contains "camera_matrix", "distortion_params", and
+            "input_image_size" (size of images used to compute these
+            parameters).
+        """
+        super().__init__(
+            images_folder=images_folder,
+            output_folder=output_folder,
+            model_path=model_path,
+            inference_settings=inference_settings,
+        )
+        self.defisheye_flag = defisheye_flag
```

Comment:
Params defisheye_flag and defisheye_params should be taken from inference_settings; now they are never updated from their default.